freeipa.spec.in

```diff
@@ -126,8 +126,8 @@ BuildRequires:  python-memcached
 BuildRequires:  python-lxml
 # 5.0.0: QRCode.print_ascii
 BuildRequires:  python-qrcode-core >= 5.0.0
-# 1.13: python-dns URI record support
-BuildRequires:  python-dns >= 1.13
+# 1.15: python-dns PY3 support (many improvements)
+BuildRequires:  python-dns >= 1.15
```

Comment:
Don't bump BuildRequires unless it is actually required for the build. Doing so does not add any value, only the chore of having to maintain additional packages in our build roots for no reason.